data/step_function_inputs/ESACCI_Biomass_L4_AGB_V4_100m_ingest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is the stpfn input file to only _ingest_ the collection : does not cogify or do data transfer.